lib/src/models/account.dart

```diff
@@ -15,13 +15,35 @@ enum AccountType {
   }
 }
 
+enum AccessAi {
+  fullAccess('FULL_ACCESS'),
+  noHistory('NO_HISTORY'),
+  noAccess('NO_ACCESS');
+
+  final String value;
+
+  const AccessAi(this.value);
+
+  /// Gets the [AccessAi] from the given value.
+  ///
+  /// If the value is not found, [noAccess] is returned.
+  static AccessAi fromValue(String? value) {
+    return AccessAi.values
+        .firstWhere((e) => e.value == value, orElse: () => AccessAi.noAccess);
```

Comment:
I am not sure if we want to make it upper case `.toUpperCase()` before compare.

Reply:
<img width="365" alt="image" src="https://github.com/user-attachments/assets/10ff09ef-4a35-43dd-94b9-b31533e93e93">

if the [API](https://linear.app/airaio/issue/PLAT-269/limit-access-ai-platform-add-support-for-explorer-app#comment-b851829b) always return upper case, it should be fine.

Reply:
![Screenshot 2024-08-20 at 9 14 02 AM](https://github.com/user-attachments/assets/af541493-a811-4d7b-aa6e-cadb70c834d2)
It looks like it's in uppercase in the [API](https://github.com/aira/platform/blob/master/core/server/AiraPlatformData/src/main/java/io/aira/data/model/Account.java)